dotCMS/src/main/java/com/dotcms/system/event/local/business/LocalSystemEventsAPIImpl.java

```diff
@@ -98,6 +99,22 @@ public boolean unsubscribe(final Class<?> eventType) {
     } // unsubscribe.
 
     @Override
+    public boolean unsubscribe(final Object subscriber) {
+
+        final AtomicBoolean unsubscribed = new AtomicBoolean(false);
+        final  Map<Class<?>, EventSubscriber> subscriberMap =
+                this.eventSubscriberFinder.findSubscribers(subscriber);
+
+        if (null != subscriberMap && !subscriberMap.isEmpty()) {
+
+            subscriberMap.forEach((eventType, eventSubscriber) ->
+                    unsubscribed.set( unsubscribed.get() | this.unsubscribe(eventType, eventSubscriber.getId()) ));
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Correct this "|" to "||". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2178)